.github/workflows/sc2-quality-assurance.yml

```
@@ -0,0 +1,237 @@
+env:
+  DOTNET_VERSION: '8.0.x'
+  CI: true
+  
+jobs:
+  sc2-integration-tests:
+    name: SC-2統合テスト・品質保証
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    
+    steps:
+    - name: チェックアウト
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)